assets/schema_input.json

```diff
@@ -8,29 +8,25 @@
         "type": "object",
         "properties": {
             "sample": {
-                "type": "string",
-                "exists": true,
+                "type": ["string", "integer"],
```

Comment:
Isn't the `exists` necessary?

Reply:
nextflow schema is migrating to the new version, so `exists` is only used for input checks now in the Nextflow schema. I’ll ping you in the Slack channel of details.

Reply:
Some updates. `"exists": true` only for file path check.